Sources/Overload/OvEditor/src/OvEditor/Panels/SceneView.cpp

```diff
@@ -199,11 +199,12 @@ void OvEditor::Panels::SceneView::HandleActorPicking()
 
 	if (m_gizmoOperations.IsPicking())
 	{
-		auto mousePosition = EDITOR_CONTEXT(inputManager)->GetMousePosition();
-
 		auto [winWidth, winHeight] = GetSafeSize();
 
-		m_gizmoOperations.SetCurrentMouse({ static_cast<float>(mousePosition.first), static_cast<float>(mousePosition.second) });
+		ImVec2 globalMousePos = ImGui::GetMousePos();
```

Comment:
I don't think we want to add direct ImGui calls to `OvEditor`. Maybe we could get the mouse position from the `InputManager`?